components/ReviewForm/index.js

```diff
@@ -7,13 +7,15 @@ import { toast } from "react-toastify";
 
 export default function ReviewForm({ activityId, onClose }) {
   const [rating, setRating] = useState(0);
+  const [comment, setComment] = useState("");
   const handleRating = (rate) => {
     setRating(rate);
   };
-
+  const handleCommentChange = (event) => {
+    setComment(event.target.value);
+  };
   const handleSubmit = async (event) => {
     event.preventDefault();
-
```

Comment:
Instead of having a state for the comment that updates on every keystroke, you can extract the data directly when submitting the form from the event object:

```js
const comment = event.target.comment.value;
```